.github/workflows/build_and_test.yml

```diff
@@ -1,14 +1,16 @@
 name: go-test
+
 on: [push]
+
 jobs:
   build_and_test:
     runs-on: ubuntu-latest
-    container: quarkchaindocker/goquarkchain
+    container: quarkchaindocker/goquarkchain:test1.6.1
```

Comment:
why labeld test1.6.1?

Reply:
This is base on mainnet1.6.1 which is the latest one and upgrade the ubuntu to 20.04 for test, as we do not have an offical release so far.